packages/teams-js/test/private/copilot.spec.ts

```diff
@@ -384,4 +387,24 @@ describe('copilot', () => {
       });
     });
   });
+
+  describe('copilot.customTelemetry', () => {
+    describe('sendCustomTelemetryData', () => {
+      it('sendCustomTelemetryData should throw if called before initialization', async () => {
+        expect.assertions(1);
+        utils.uninitializeRuntimeConfig();
+        await expect(copilot.customTelemetry.sendCustomTelemetryData(new UUID())).rejects.toThrowError(
+          new Error(errorLibraryNotInitialized),
+        );
+      });
+
+      it('sendCustomTelemetryData message should not be null', async () => {
```

Comment:
>should not be null [](http://example.com/codeflow?start=42&length=18)

Should this test validate that the contents of the message is actually correct and not just non-null (e.g., contains the provided UUID?)

Reply:
Yes, it should. I tried adding some unit tests, but they require a bit more time to do correctly. I propose adding them in a subsequent change so that this PR can make the next release.